src/pages/auth/error.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
+
+import { NextPage } from "next";
+import { useRouter } from "next/router";
+
+import { Button, Center, Code, Flex, Heading, Text, useToast } from "@chakra-ui/react";
+
+interface Props {
+  error?: string | string[]
+}
+
+const Error: NextPage<Props> = ({ error }) => {
+  const router = useRouter()
+  const toast = useToast()
+
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    if (error && error == 'AccessDenied') {
+      toast({
```

Comment:
pode ser criado uma pasta de utils com essas funções de erro prontas para alterar só a mensagem/título, considerando que há inúmeras linhas de configuração de estilo que podem ser poupadas!